scene/animation/animation_node_state_machine.cpp

```diff
@@ -877,13 +883,15 @@ void AnimationNodeStateMachine::_rename_transitions(const StringName &p_name, co
 		}
 
 		if (transitions[i].to == p_name) {
-			Vector<String> path = String(transitions[i].from).split("/");
+			Vector<String> path = String(transitions[i].to).split("/");
```

Comment:
I don't think this change makes sense since this is to get the path of the parent of the nested state machine.